source/module_base/abfs-vector3_order.cpp

```diff
@@ -27,10 +27,10 @@ template<>
 bool operator< ( const Abfs::Vector3_Order<int> &v1, const Abfs::Vector3_Order<int> &v2 )
 {
 	if      ( v1.x < v2.x ) return true;
-	else if ( v1.x > v2.x ) return false;
+	if ( v1.x > v2.x ) return false;
```

Comment:
I feel the logic has been changed here.

Reply:
There's no need to use `else` after `return`, see https://clang.llvm.org/extra/clang-tidy/checks/readability/else-after-return.html